### ISSUE TITLE
Setting up MainActivity.java

### ISSUE THREAD

**Issue by luandro** [axemclion/react-native-cordova-plugin#21]
It seems the react-native's files have changed a bit. I've been trying to get this to work with a clean React-Native install, it all makes sense until I get to the setup for the MainActivity.java file. 

This is what it looks like.

```
public class MainActivity extends ReactActivity {
   ...
  @Override
    protected List<ReactPackage> getPackages() {
        return Arrays.<ReactPackage>asList(
            new MainReactPackage()
        );
    }
}
```

This is what I tried changing it to, based on what I understood from the README:

```
public class MainActivity extends io.cordova.reactnative.CordovaActivity {
   ...
   @Override
    protected List<ReactPackage> getPackages() {
        return Arrays.<ReactPackage>asList(
            new MainReactPackage(),
            super.cordovaPluginPackage
        );
    }
```

But I'm getting an error:

```
MainActivity.java:16: error: method does not override or implement a method from a supertype
    @Override
    ^
```

What am I doing wrong? Could the README be updated to reflect the new default code in React-Native?

**Comment by axemclion** [comment]
@luandro I am working on this right now. You can look at the `examples` branch to see how it will work.
I have also updated the settings into the `dev` branch. I am in the middle of testing, will push out an update tomorrow.

Maybe a duplicate of https://github.com/axemclion/react-native-cordova-plugin/issues/14

**Comment by luandro** [comment]
Awesome, thanks for the hard work. I'll close since it's a duplicate.

**Comment by axemclion** [comment]
:+1:

**Comment by axemclion** [comment]
@luandro Just published 1.0.0 with more examples and documentation

**Comment by luandro** [comment]
Awesome, thank you. I'll check it out once I have some time. Cheers!
